backend/authors/models.py

```diff
@@ -0,0 +1,11 @@
+import uuid
+from django.db import models
+
+class Author(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    host = models.URLField(max_length=200)
+    url = models.URLField(max_length=200, default=str(host)+ str(id))
```

Comment:
i thought this worked here, but turns out it doesn't really, the correct way to do this in django is to define a function in this class like the following below:
"""
def update_url(self):
    self.url = self.host + "/" + self.id
    self.save()
 """